practices/practice-one/src/scripts/controllers/todo.controller.ts

```diff
@@ -0,0 +1,86 @@
+import { TodoType } from '../models/todo.model'
+import { Param } from '../views/templates/todo-item.template'
+import store from '../helpers/store';
+import TYPE from '../constants/type-message.constant';
+import TodoFormView from '../views/todo-form.view';
+import TodoAction from '../constants/hash.constant';
+
+type Delete = {
+    type: string;
+    dataLength: number;
+}
+
+export default class TodoController {
+    private todos: Array<TodoType>;
+    private todoView: TodoFormView;
+    constructor(TodoFormView) {
+        const todoLocals = JSON.parse(store('todos').get());
+        this.todos = todoLocals || [];
+        this.todoView = TodoFormView
+    }
+
+    getTodos(hash: string): Array<TodoType> {
+        switch (hash) {
+            case TodoAction.COMPLETED: {
+                return this.todos.filter((todo) => todo.isCompleted);
+            }
+            case TodoAction.ACTIVE: {
+                return this.todos.filter((todo) => !todo.isCompleted);
+            }
+            default: {
+            }
+        }
+        return this.todos;
+    }
+
+    handleAddTodo(text: string) {
+        if (Boolean(text)) {
+            const date = new Date();
+            const currentDate = `${date.getDate()}-${date.getMonth() + 1}-${date.getFullYear()}`
+            const data: TodoType = {
+                id: this.todos.length,
+                description: text,
```

Comment:
Why not title?

Reply:
I thought it was the content so I put it like that

Reply:
- I will update it soon